spec/cucumber/formatter/interceptor_spec.rb

```diff
@@ -20,7 +14,7 @@ module Cucumber::Formatter
 
       context 'when passed :stderr' do
         before :each do
-          @stderr = $stdout
+          @stderr = $stderr
```

Comment:
We should save `$stderr` so we can put it back how we found it after the spec run.

Reply:
We don't need an each declaration when we're inside an area with one test. But that's probably splitting hairs.